bin/build_container.sh

```diff
@@ -1,53 +0,0 @@
-#!/bin/bash
-
-set -e
-
-THIS_DIR=$(dirname $(readlink -f $0))
-PROJ_ROOT=${THIS_DIR}/..
-BASE_DIR=${PROJ_ROOT}/../..
-
-IMAGE_NAME="experiment-lammps"
-VERSION="$(< ${BASE_DIR}/VERSION)"
-FAASM_VERSION="$(< ${BASE_DIR}/faasm/VERSION)"
-
-pushd ${PROJ_ROOT} >> /dev/null
-
-export DOCKER_BUILDKIT=1
-
-# Docker args
-if [ "$1" == "--no-cache" ]; then
-    NO_CACHE=$1
-else
-    NO_CACHE=""
-fi
```

Comment:
This is sort of a fake way to do bash script arguments (i.e. just string matching). If you need to do it, you can use `getopts`, however, we should write things like this using a Python CLI wherever possible.